packages/web-components/fast-components/src/calendar/calendar.styles.ts

```diff
@@ -0,0 +1,65 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+import {
+    accentForegroundActiveBehavior,
+    heightNumber,
+    neutralForegroundRestBehavior,
+} from "../styles/index";
+
+export const CalendarStyles = css`
+    ${display("block")} :host {
+        --cell-height: calc(${heightNumber} * 1px);
+        --weekday-background: transparent;
+        --weekday-color: inherit;
+        --current-day: ${accentForegroundActiveBehavior.var};
+        --current-day-border: none;
+        --current-day-background: none;
+        --inactive-day-color: ${SystemColors.GrayText};
+    }
+
+    .title {
+        font-size: 1.5em;
+        padding: 5px 0;
+        text-align: center;
+    }
+
+    .days {
+        border-left: var(--cell-border);
+        border-top: var(--cell-border);
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```

Comment:
Just FYI in case you weren't aware, you can write this as `repeat(7, 1fr)` to condense and make this a little easier to understand at a glance.